web-client/src/environments/environment.bhutan-demo.ts

```diff
@@ -5,8 +5,8 @@ export const environment: Environment = {
 
   // Enable persistence for demo purposes.
   persistAkitaState: true,
-  nautilusWalletServer: 'https://wallet-bhutan-demo-api.ntls.io/',
-  nautilusAssetServices: 'https://wallet-bhutan-demo-services.ntls.io/',
+  nautilusWalletServer: 'https://wallet-bhutan-api.ntls.io/',
+  nautilusAssetServices: 'https://wallet-bhutan-services.ntls.io/',
```

Comment:
https://github.com/ntls-io/nautilus-wallet-demo/blob/e33004eb60eea710989f24aaf0f47789e1b86279/docker-compose.yaml#L158

Do we need to fix the `VIRTUAL_HOST` in `asset-services-bhutan-demo-api`?

Reply:
Yep, I have a fix for that as well.  I already have the branch, just need to make the PR

Reply:
Ok then this PR is good to ship.